src/hardware/mpu401.cpp

```diff
@@ -780,16 +780,17 @@ class MPU401 final : public Module_base {
 	        : Module_base(configuration),
 	          installed(false)
 	{
-		if (!MIDI_Available())
-			return;
+		Section_prop* section = dynamic_cast<Section_prop*>(configuration);
+		assert(section);
 
-		Section_prop *section = static_cast<Section_prop *>(configuration);
-		const auto mpu_type = std::string(section->Get_string("mpu401"));
-		if (mpu_type == "none" || mpu_type == "off" || mpu_type == "false")
-			return;
+		const std::string_view mpu_type = section->Get_string("mpu401");
+
+		installed = (mpu_type != "none" && mpu_type != "off" &&
+		             mpu_type != "false");
```

Comment:
This misses "disabled" and "no".

I suggest factoring out the boolean string checks into helper functions from `Value::SetBool` in `setup.cpp`, then use that here (e.g. `parse_bool_setting(mpu_type)`).

`std::optional<bool> parse_bool_setting()` would be probably the best; then it's the client's call whether to raise  and error on invalid values, or just default to whatever the default is (could be either "enabled" or "disabled"; it would be a mistake to always default to "disabled", so "false").